www/package.json

```diff
@@ -93,7 +93,7 @@
   "scripts": {
     "build": "gatsby build",
     "deploy": "gatsby build --prefix-paths && gh-pages -d public",
-    "develop": "gatsby develop",
+    "develop": "GATSBY_GRAPHQL_IDE=playground gatsby develop",
```

Comment:
hahah 👍

Reply:
I don't mind having this in - especially if we want to move forward with playground, using it for `gatsbyjs.org` dev seems actually good idea